Dockerfile

```diff
@@ -25,23 +25,41 @@
 #     -i -t --rm \
 #     --runtime=nvidia \
 #     core3d/danesfield \
-#     danesfield/tools/material_classifier.py --cuda ...
+#     danesfield/tools/material_classifier.py --cuda ...    
+
+FROM nvidia/cuda:10.0-devel-ubuntu18.04
 
-FROM nvidia/cuda:9.0-runtime-ubuntu16.04
 LABEL maintainer="Max Smolens <max.smolens@kitware.com>"
 
+COPY patches /patches
+
 # Install prerequisites
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    bzip2 \
-    ca-certificates \
-    curl \
-    libgl1-mesa-glx \
-    libglu1-mesa \
-    libxt6 \
-    xvfb \
-    sudo && \
-    apt-get clean -y && \
-    rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y software-properties-common && apt-get update && \
+  add-apt-repository -y ppa:ubuntu-toolchain-r/test && add-apt-repository -y ppa:ubuntugis/ppa && apt-get update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+  sudo \
+  make \
+  git \
+  bzip2 \
+  ca-certificates \
+  curl \
+  libgl1-mesa-glx \
+  libglu1-mesa \
+  libxt6 && \
+  apt-get clean -y && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN git clone --recursive https://github.com/Kai-46/ColmapForVisSat.git && \
```

Comment:
Unless they need to be in this order,  I would recommend moving this `git clone` down later to right before `ColmapForVisSat/ubuntu1804_install_dependencies.sh` is called.